docs/application/vstools/tools/project-wizard.md

```diff
@@ -31,10 +31,12 @@ When you create a new project with a specific template, the Project Wizard autom
 To create a Tizen .NET application project:
 
 1.  To open the Project Wizard:
+
     a.  In the Visual Studio menu, go to **File &gt; New &gt; Project &gt; Templates &gt; Visual C\# &gt; Tizen**.
+
```

Comment:
Please change a, b, and c to 1, 2, and c. 
Markdown doesn't support type attribute.

Reply:
Done.